src/effiara/annotator_reliability.py

```
@@ -19,76 +20,111 @@ class Annotations:
+        self.num_classes = label_generator.num_classes
+        self.annotation_col_suffixes = self.label_generator.label_suffixes
+
         self.agreement_metric = agreement_metric
+        self.agreement_suffix = agreement_suffix
+        self.agreement_type = agreement_type
+
+        self.overlap_threshold = overlap_threshold
+
         self.merge_labels = merge_labels
 
-        # Populates self.annotators
-        # by looking at the *_label columns in self.df
-        self.label_generator = label_generator
-        if label_generator is None:
-            self.label_generator = DefaultLabelGenerator.from_annotations(
-                self.df, num_classes)
-        self.annotators = self.label_generator.annotators
-        self.num_classes = self.label_generator.num_classes
-        self.label_mapping = self.label_generator.label_mapping
+        self.reliability_alpha = reliability_alpha
 
-        self.replace_labels()
+        # TODO: add some check for reannotations
+        self.reannotations = reannotations  # TODO: replace w/ function to look for re_
+
+        # set in self.calculate_inter_annotator_agreement
         self.overall_inter_annotator_agreement = np.nan
+
+        # merge labels
+        self.replace_labels()
+
+        # generate annotation labels
+        self.df = self.label_generator.add_annotation_prob_labels(self.df)
+
+        # calculate agreements
         self.G = self.init_annotator_graph()
         self.calculate_intra_annotator_agreement()
         self.calculate_inter_annotator_agreement()
-        self.calculate_annotator_reliability()
+        self.calculate_annotator_reliability(alpha=self.reliability_alpha)
 
     def replace_labels(self):
         """Merge labels. Uses find and replace so do not switch labels e.g.
         {"misinfo": ["debunk"], "debunk": ["misinfo", "other"]}.
         """
-        if self.merge_labels is None:
+        if not self.merge_labels:
```

When comparing to None, PEP 8 says to use is not None (https://peps.python.org/pep-0008/#programming-recommendations). If you want to include the empty dict in this check I suppose this would work, but I always feel its best to be explicit about what you're checking. In that case it might be better to either raise an error (e.g., if not self.merge_labels: raise ValueError("merge_labels should not be empty")) or just let it be because the for loop on line 112 won't do anything in this case.